topi/include/topi/nn/upsampling.h

```diff
@@ -55,7 +55,7 @@ inline Tensor upsampling(const Tensor& input,
                          std::string mode = "NEAREST_NEIGHBOR",
                          std::string name = "tensor",
                          std::string tag = kInjective) {
-  return resize(input, shape, layout, false, mode);
+  return resize(input, shape, layout, true, mode);
```

Comment:
This change should be removed. (I know it is no longer used)

Reply:
You're right, I've made this change along with reverting some of the changes I made to frontends that are no longer needed due to defaulting upsampling to align_corners=False.